### PR TITLE
CSEN-30 User DAO integration tests were added

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

File: src/it/scala/com/bwsw/cloudstack/entities/UserDaoIntegrationTestSuite.scala [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  val secondUserId = UUID.randomUUID()
+
+  it should "retrieve empty user list" in {
+    val userFindRequest = new UserFindRequest().withId(firstUserId)
+    val initUsers = userDao.find(userFindRequest)
+
+    assert(initUsers.isEmpty)
+  }
+
+  it should "retrieve user after it creation" in {
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/entities/UserDaoIntegrationTestSuite.scala]: after its creation

File: src/it/scala/com/bwsw/cloudstack/entities/UserDaoIntegrationTestSuite.scala [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    )
+    val firstUserCreateRequest = new UserCreateRequest(firstUserCreationSettings).withId(firstUserId)
+    userDao.create(firstUserCreateRequest)
+
+    val userFindRequest = new UserFindRequest().withId(firstUserId)
+    val updatedUsers = userDao.find(userFindRequest)
+    assert(updatedUsers.size == 1 && updatedUsers.head.id == firstUserId)
+  }
+
+  it should "retrieve all created users" in {
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/entities/UserDaoIntegrationTestSuite.scala]: it's not good that this test and the previous one depend on each other

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/cloudstack/entities/UserDaoIntegrationTestSuite.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/entities/UserDaoIntegrationTestSuite.scala]: ok

File: src/it/scala/com/bwsw/cloudstack/entities/UserDaoIntegrationTestSuite.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/entities/UserDaoIntegrationTestSuite.scala]: combined into one test

**Comment by MedvedevBW** [comment]
@placccebo

**Review by placccebo — APPROVED** [review]

**Pull request merged**